Iyy banner video

joetower opened the pull request:
## [YALB-XX: Title](https://yaleits.atlassian.net/browse/YALB-XX)

### Description of work
- Adds functionality bullet item
- Fixes this or that bullet item

### Functional testing steps:
- [ ] Step 1
- [ ] Step 2
- [ ] Step 3
- [ ] ...

miketullo95 commented:
@joetower we got some feedback on this block from OPAC. I know this doesnt have a ready for review tag, but I didnt want to forget these. Also im attaching a block icon 😄 

- [ ] could we have the 'full' dial have the video touching the left and right side like how grand hero does?

![Video Banner](https://github.com/user-attachments/assets/4620e544-2659-47ee-abd2-53441feee819)

joetower commented:
@miketullo95 I have addressed your feedback in a new PR #863 and will be closing this PR because it was branched from `iyy` and we need it in `develop`. 

I will be adding testing details on Monday and open it along with the Atomic and Component Library PRs. I'll let you know when it's ready to test properly.
